tests/specs/swap/swap.spec.ts

```diff
@@ -2,7 +2,8 @@ import { test } from '../../fixtures/fixtures';
 
 const hiroApiPostRoute = '*/**/v2/transactions';
 
-test.describe('Swaps', () => {
+// Skip as swaps feature is disabled
```

Comment:
Are we able to skip this based on config?

Reply:
not sure here